isisd/isis_zebra.c

```diff
@@ -708,6 +708,43 @@ int isis_zebra_label_manager_connect(void)
 	return 0;
 }
 
+bool isis_zebra_srv6_manager_ready(void)
+{
+	return (zclient_sync->sock > 0);
+}
+
+/**
+ * Connect to the SRv6 Manager.
+ *
+ * @return	0 on success, -1 otherwise
+ */
+int isis_zebra_srv6_manager_connect(void)
+{
+	/* Connect to label manager. */
+	if (zclient_sync->sock <= 0) {
```

Comment:
you mean sid manager ? comment is wrong.

Reply:
@pguibert6WIND 

Thanks for the review.

Based on the feedback of the community, I have updated the SID Manager to make the API asynchronous. So, this function no longer exists.